### PR TITLE
Test OSSF scorecard action

### PR DESCRIPTION
**Pull request by stone-z**
### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] (Giant Swarm) If creating a release, bump the `version` and `appVersion` in Chart.yaml.

### CODE REVIEW

**Comment by stone-z** [comment]
Ok, apparently this can't run from a branch so will have to try it from main

**Comment by fhielpos** [comment]
I think once it's on main you can run it from a branch

File: .github/workflows/test-scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      id-token: write
```
Comment (stone-z) [inline .github/workflows/test-scorecard.yml:29]:
```suggestion
      # id-token: write
```

**Comment by fhielpos** [comment]
You are running something though:
https://github.com/giantswarm/starboard-exporter/actions/runs/7211528041/job/19647173885

**Pull request merged**